rotkehlchen/data_migrations/migrations/migration_2.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import TYPE_CHECKING
+
+import gevent
+
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.utils.mixins.common import function_sig_key
+
+if TYPE_CHECKING:
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
```

Comment:
Logger is not used here unless I'm blind

Reply:
No it's not. Will remove.